supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -45,9 +49,12 @@
 class EmailVerificationRecipe(RecipeModule):
     recipe_id = 'emailverification'
     __instance = None
+    email_delivery: EmailDeliveryIngredient[TypeEmailVerificationEmailDeliveryInput]
 
     def __init__(self, recipe_id: str, app_info: AppInfo,
-                 config: ParentRecipeEmailVerificationConfig):
+                 config: ParentRecipeEmailVerificationConfig,
+                 ingredients: Union[EmailVerificationIngredients, None] = None
```

Comment:
Do you want to call it `email_ingredient` ? We will have sms ingredient as well in few recipes

Reply:
`EmailVerificationIngredients` can contain `EmailDeliveryIngredient` as well `SMSDeliveryIngredient`. This is the same as the node PR.